maestro-client/src/main/java/maestro/android/AndroidBuildToolsDirectory.kt

```diff
@@ -0,0 +1,60 @@
+package maestro.android
+
+import java.io.File
+import java.util.regex.Pattern
+
+internal object AndroidBuildToolsDirectory {
+
+    fun findBuildToolsDir(androidHome: File): File {
+        val buildToolsParent = File(androidHome, "build-tools")
```

Comment:
This makes Maestro dependent on Android SDK, contrary to what we've been saying. Let's make sure this is not a required step for running a simple flow on an already debuggable app

Reply:
Good point. I'll add some logic that detects whether the app is already debuggable and if so, `ApkDebuggable.enable` can just copy the src apk to dst location.

Reply:
Awesome! 👍